rules/aip0133/resource_reference_type.go

```diff
@@ -26,12 +26,7 @@ import (
 var resourceReferenceType = &lint.MethodRule{
 	Name: lint.NewRuleName(133, "resource-reference-type"),
 	OnlyIf: func(m *desc.MethodDescriptor) bool {
-		// Return type of the RPC.
-		ot := m.GetOutputType()
-		if ot.GetName() == "Operation" {
-			ot = utils.GetOperationResponseType(m)
-		}
-
+		ot := utils.GetResponseType(m)
```

Comment:
If you are doing this fix in this PR, it is also in rules/aip0233/resource_reference_type.go :)

Reply:
done! fixed in a couple places.